Test.hs

```diff
@@ -419,6 +460,10 @@ test_glfwWaitEvents :: IO ()
 test_glfwWaitEvents =
     c'glfwWaitEvents
 
+test_glfwWaitEventsTimeout :: IO ()
+test_glfwWaitEventsTimeout =
+    c'glfwWaitEventsTimeout 1.0
```

Comment:
I think it will make the test one second longer, are you ok with changing this value to 0.001 ?